src/error.rs

```diff
@@ -207,9 +207,7 @@ pub enum StandardIoAction {
 
 impl StandardIoAction {
     fn text(&self) -> &'static str {
-        match self {
-            StandardIoAction::Read => "read from",
-        }
+        "read from"
```

Comment:
Undo this please so that when new variants are added we get a type error and the code gets correctly updated.

Reply:
✅